moped-editor/src/utils/mapHelpers.js

```diff
@@ -735,10 +708,9 @@ export function useHoverLayer() {
       setFeatureId(null);
       return;
     }
-
     // Otherwise, get details for tooltip
     const {
-      srcEvent: { offsetX, offsetY },
+      originalEvent: { offsetX, offsetY },
```

Comment:
`srcEvent` is no longer exposed by `react-map-gl` so the offset has to come from `originalEvent` now